### ISSUE TITLE
Linter warnings are handled / shown as error

### ISSUE THREAD

**Issue by strangelookingnerd** [camunda/camunda-modeler-custom-linter-rules-plugin#37]
Not sure if this is the right place or if this should be moved to https://github.com/camunda/camunda-modeler instead. Just let me know ;)

I'm trying to implement custom rules. My plugin and rules used to work just fine in Camunda Modeler < 5.3 with the "old" approach.
Now I updated the modeler and adapted to the build-in linter plugin and the way custom rules are now implemented.

I have custom rules that issue a WARNING (as opposed to an error). See `index.js`

```
module.exports = {
  configs: {
    recommended: {
      rules: {
        'end-event-listener': 'error',
        'error-sub-process': 'error',
        'report-activity': 'warn',
        'configuration-activity': 'warn',
      }
    }
  }
}
```

As you can see the "overall" number of errors is 4 in the bottom status bar - opening the linter tab shows 6 errors including two of my custom rules that should be warnings.
![image](https://user-images.githubusercontent.com/49242855/205100331-8781cee5-6e27-41b5-8183-d0df18771b3d.png)

Camunda Modeler 5.5.1 and Plugin based on current HEAD of this repo.

**Comment by nikku** [comment]
Thanks for opening this issue.

The situation will improve with 5.6 where we add full support to lint warnings (currently shown as errors, as you reported). No plug-in API changes so you should be able to sit back, upgrade and see the editor properly differentiate errors (and warnings).
